DPPP/MSWriter.cc

```diff
@@ -283,7 +284,13 @@ namespace DP3 {
         }
       }
       // Remove possible hypercolumn definitions.
+// Test for casacore version 3.1.1 or smaller
+#pragma message "BOOST_VERSION=" BOOST_PP_STRINGIZE(CASACORE_VERSION_NUM)
```

Comment:
Shouldn't you have removed this (debug) line?

In any case, it's the Casacore version, not the boost version ;)

Reply:
Oops. Thanks. I was a bit too quick with the merge. Will remove the line.